app/assets/javascripts/thredded/core/on_page_load.es6

```diff
@@ -19,31 +18,29 @@
   // has already fired, will execute the callback immediately.
   window.Thredded.onPageLoad = (callback) => {
     pageLoadCallbacks.push(callback);
-    // With async script loading, a callback may be added
-    // after the DOMContentLoaded event has already triggered.
-    // This means we will not receive turbolinks:load on Turbolinks 5.
-    if (isTurbolinks5 && !onPageLoadFiredOnce && window.Thredded.DOMContentLoadedFired) {
+    // With async script loading, a callback may be added after the DOMContentLoaded event has already triggered.
+    // This means we will receive neither a DOMContentLoaded event, nor a turbolinks:load event on Turbolinks 5.
+    if (!onPageLoadFiredOnce && window.Thredded.DOMContentLoadedFired) {
       callback();
     }
   };
 
-  if (!isTurbolinks || isjQueryTurbolinks) {
-    $(() => triggerOnPageLoad());
-    return;
-  }
-
   if (isTurbolinks5) {
     document.addEventListener('turbolinks:load', () => {
       triggerOnPageLoad();
     });
   } else {
-    // Turbolinks Classic with no jQuery.Turbolinks:
-    $(() => {
-      triggerOnPageLoad();
+    // Turbolinks Classic (with or without jQuery.Turbolinks):
```

Comment:
this comment is wrong  -- this stanza represents Turbolinks Classic (with or without jQuery.Turbollinks) **or no Turbolinks**